.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,35 @@
+name: SonarCloud
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
Please limit to push only on root branch (`develop`).  
As this test takes 5 minutes, it *will* use unneeded resources.

Reply:
Good idea. I forgot to switch back to only on develop after testing the CI.

Reply:
I don't see why the branch `ci-test` is needed. But alright.

Reply:
For testing purposes if CI-pipeline needs to be tested without messing with the git history and need squashing